src/__tests__/upsert.test.ts

```diff
@@ -4,7 +4,7 @@ import clients from '../clients';
 import main from '../main';
 import { TestInputs, clearInputs, getBaseInput, getInput, setInputs } from './utils/utils';
 
-describe('Main Integration Tests', () => {
+describe('Upsert Integration Tests', () => {
```

Comment:
Can you add one test that validates the default behaviour of UPSERT?

Reply:
We prefer to configure the default operation in `action.yml`, therefore we can test it only through github action.